Update hosts.example

julienlim opened the pull request:
@mbukatov @r0h4n @shirshendu 

Added some additional vars (etcd_tls_client_auth=true and configure_firewalld_for_tendrl) that can be set along with documentation on how to set it.

Note: tendrl-vagrant doesn't work out of the box with its current firewalld setting, so for sandboxes, disabling configure_firewalld_for_tendrl is sometimes useful.

This also addresses partly one of the comments made in https://github.com/Tendrl/documentation/issues/104#issue-337508606.

julienlim committed:
Update hosts.example

Added some additional vars (etcd_tls_client_auth=true and configure_firewalld_for_tendrl) that can be set along with documentation on how to set it.

Note: tendrl-vagrant doesn't work out of the box with its current firewalld setting, so for sandboxes, disabling configure_firewalld_for_tendrl is sometimes useful.

mbukatov commented on hosts.example:
```diff
@@ -29,5 +29,14 @@ graphite_fqdn=tendrl.example.com
 
+# Uncomment to configure etcd to use client to server authentication with HTTPS 
+# client certificates, as well as all Tendrl components will be reconfigured 
+# accordingly.  Default set to false.
```
For this comment to be fully clear, we will need to add much more information there.

julienlim replied:
@mbukatov Both of these variables are already mentioned in the README.

The readme pretty much says "etcd tls client authentication" for the etcd_tls_client_auth var.

mbukatov commented on hosts.example:
```diff
@@ -29,5 +29,14 @@ graphite_fqdn=tendrl.example.com
+#etcd_tls_client_auth=true
+
+# Uncomment to configure to not use of firewalld on the tendrl_server and 
+# gluster_servers.  Default set to true.
```
For this comment to be fully clear, we will need to add much more information there.

julienlim replied:
@mbukatov The readme says "firewalld setup for Tendrl (variable configure_firewalld_for_tendrl)" for the firewalld configuration.

Let me update the comments for the host inventory file and see if the revised version is easier to understand.

julienlim replied:
I've revised this PR -- see https://github.com/Tendrl/tendrl-ansible/pull/110.  We can go ahead and not delete / archive this PR.

mbukatov reviewed (commented):
If we really want to list these 2 particular variables in the example inventory file, we would need either duplicate the full description of the variable from the readme file or reference it there.

I will think about minimal tweaks we need to do here and suggest changes so that we can merge this soon.

julienlim committed:
Another host.example update

Updated per comments and also added another variable tendrl_copr_repo so folks know they can optionally use master (and not just released bits).

julienlim commented:
@mbukatov I updated hosts.example and improve documentation on the additional ansible vars.

I reviewed the readme (in a few places) and tried to be more specific and also align the comments with how the var is mentioned in the readme.

Per your request, I've updated the original PR and closed the other PR (https://github.com/Tendrl/tendrl-ansible/pull/110) I created.

julienlim committed:
Merge branch 'master' into patch-1

mbukatov commented:
Since we are unlikely to need  any new variables, the bit of duplication is no longer a problem. Merged.
